ADALiOS/ADALiOS/ADAuthenticationContext+TokenCaching.m

```diff
@@ -93,7 +94,7 @@ - (ADTokenCacheStoreItem*)findCacheItemWithKey:(ADTokenCacheStoreKey*) key
         else
         {
             //We have a cache item that cannot be used anymore, remove it from the cache:
-            [self.tokenCacheStore removeItemWithKey:key userId:userId error:nil];
+            [self.tokenCacheStore removeItemWithKey:key userId:userId.userId error:nil];
```

Comment:
Does it create multiple entries in cache if first request has UserIdentifier's type UniqueId and second request has RequiredDisplayableId? Both of them should refer to same cache entry since it is same user.